eda-env/Lib/site-packages/pandas/tests/window/test_expanding.py

```
@@ -0,0 +1,723 @@
+                ({"A": [1, np.nan], "B": [np.nan, 5]}, [0, 1]),
+                ({"A": [1, np.nan, 3], "B": [np.nan, 5, 6]}, [0, 1, 2]),
+            ],
+            1,
+        ),
+    ],
+)
+def test_iter_expanding_dataframe(df, expected, min_periods):
+    # GH 11704
+    expected = [DataFrame(values, index=index) for (values, index) in expected]
+
+    for expected, actual in zip(expected, df.expanding(min_periods)):
```

Ruff (0.11.9)

Loop control variable `expected` overrides iterable it iterates

(B020)